api/api.py

```diff
@@ -0,0 +1,23 @@
+import httpx
+from typing import Dict
+
+async def call_api(node_name: str, payload: Dict) -> Dict:
+    """
+    Function to call an API with a given URL, payload, and node name.
+
+    Args:
+        node_name (str): The name of the node to include in the URL.
+        payload (Dict): The data to send with the request.
+
+    Returns:
+        Dict: The response from the API.
+    """
+    # Format the URL to include the node_name
+    api_url = f"http://localhost:8000/api/v1/private/applications/endpoint/{node_name}"
```

Comment:
We should not hard code url. Depending on environment it will change. There will be .env, production.env, staging.env files from which we should get the url. And it should contain the application_id, without that we will not know which application is trying to get executed. @suchana-aiplanet